frontend/components/layout/StagingHeader.vue

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
Can you please rename this component to `StagingBanner`? Since placing it near actual `Header` suggests that it's a different version of it